New nav structure

vgantchev opened the pull request:
New nav structure

vgantchev commented:
@jak-pan @SimonKraus

SimonKraus commented:
i'm okay with this approach. 
Only thing is i would name the  "Snakenet" category "How-to" because most articles are  still valid after snakenet. 
Maybe putting snakenetas  subsection of how-to.
We should do this  quick as  we have many translations ready to be published

jak-pan commented:
Yeah nominating and validating guides will pass to mainnet. I'd not get them under a subsection if we're changing stuff. Maybe just make snakenet a single article under learn?

SimonKraus commented:
@vgantchev i talked to jakub and we agreed not to nest the documents in the filesystem for various reasons.
we do not want to deal with redirects
we need to be able to reorganize nav without messing around with redirects and file locations

so the documents will stay in the single directory (don't worry, we'll find what to edit).

We just edit the nav structure.

So we can do the following categories in the main nav:
"How-to" with claim, become a nominator, become a validator, connect to a site
"Learn" with Staking and Snakenet
"Maintain" with node-setup, performance bench, node monitoring
"Build" <= can be added later when content is available

@jak-pan please take a look once again and confirm :pray:

jak-pan commented:
Yeah, the flat structure has multiple positives.
- It's easier to i18n
- If you update menu structure because it's no longer relevant (which happens a lot), you don't need to change all of the links and update every file, including i18n.
- You will never need to make redirects
- It's harder to duplicate stuff in a large knowledge base, everything is in the same place.

Now it does have some disadvantages but it's easy to overcome them by having proper on-point names for the files, using edit button to find the doc you're searching for, or using search of the editor you are using.

SimonKraus commented:
@vgantchev  let me know if i should apply the new nav structure

jak-pan commented on .gitignore:
```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 .docusaurus
-build
```
build/

(don't want to commit build files)

vgantchev replied:
yeah you are right, I will rename build.md

vgantchev commented:
Thanks for the suggestions guys! I changed the navs and removed the file tree. For now I think we can have Snakenet under Intro, as both are not very informative (i will pick them up soon though)

jak-pan commented on docs/build.md:
```diff
@@ -0,0 +1,8 @@
+---
+id: build
```
Probably don't need this unless we have something?
